tests/model.test.ts

```diff
@@ -212,4 +215,24 @@ describe("Model", () => {
     const model = new Model(modelData);
     expect(model.exportData()).toMatchSnapshot();
   });
+
+  test("snapshot when importing xlsx file", async () => {
+    const originalWarn = console.warn.bind(console.warn);
+    console.warn = () => {};
```

Comment:
use can use `jest.spyOn` :wink: 
```suggestion
    jest.spyOn(console, "warn").mockImplementation(() => {});
```